new-pipeline/src/dashboards.js

```diff
@@ -1073,16 +1073,17 @@ function buildDictionaryURL(metric, channel, description) {
 
 function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = buildDictionaryURL(metric, channel, description);
-  var descr = metric;
```

Comment:
`descr` is still used below, we should leave a `var descr;` line here.
(Otherwise `descr` becomes a [global variable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#Description).)

Reply:
I seem to have missed that global variable point. Thank you for the suggestion. :)